internal/clients/composer/client.go

```diff
@@ -38,23 +35,20 @@ type ComposerClientConfig struct {
 	ClientId     string
 	OfflineToken string
 	ClientSecret string
-}
-
-type tokenResponse struct {
-	AccessToken string `json:"access_token"`
+	Tokener      oauth2.Tokener
 }
```

Comment:
The same comment. You could create config without tokener, but it is useful for better testing.